docs/guides/oracle-operator-manual.md

```diff
@@ -1,143 +1,135 @@
+The oracle source code is available at https://github.com/lidofinance/lido-oracle.
 
-The algorithm of the above oracle implementation is simple and each step of an infinite loop can be broken down into two sub-steps: update beacon data and update stETH price data.
+Modules fetch the reportable slot, and if this slot is finalized, calculate and send the report to AccountingOracle and ExitBusOracle smart contracts.
 
-Update Beacon Data
+### Environment variables
 
-The daemon fetches the reportable epoch from the `LidoOracle` contract, and if this epoch is finalized on the Beacon chain, pushes the data to the `LidoOracle` contract by submitting a transaction. The transaction contains a tuple:
+The oracle daemon requires the following environment variables:
 
-```text
-(
-  epoch,
-  sum_of_balances_of_lido_validators,
-  number_of_lido_validators_on_beacon
-)
-```
+**Required**
+- `EXECUTION_CLIENT_URI` - list of Execution Client uris separated with comma. The second and next uris will be used as fallback.
+- `CONSENSUS_CLIENT_URI` - list of Consensus Client uris separated with comma. The second and next uris will be used as fallback.
+- `KEYS_API_URI` - list of Key API client uris separated with comma. The second and next uris will be used as fallback.
+- `LIDO_LOCATOR_ADDRESS` - Lido Locator smart contract address.
 
-Keep in mind that some of these transactions may revert. This happens when a transaction finalizing the current frame gets included in a block before your oracle's transaction. For example, such a transaction might had already been submitted by another oracle (but not yet included in a block) when your oracle fetched the current reportable epoch.
+**Optional**
 
-Update stETH Price Data
+**One of:**
+- `MEMBER_PRIV_KEY` - Private key of the Oracle member account.
+- `MEMBER_PRIV_KEY_FILE` - A path to the file contained the private key of the Oracle member account.
 
-The daemon checks the time-shifted price of stETH token in Curve ETH/stETH pool and evaluates how much this price differs from the current `StableSwapOracle` state. If the difference exceeds the threshold set in the `StableSwapOracle` contract, the daemon generates offchain proof for the new stats and sends it to the contract. The contract validates the proof and records new stETH stats onchain.
-This transaction can also fail in the case when another Lido oracle submits the updated state between the check and transaction submission.
+Full list could be found [here](https://github.com/lidofinance/lido-oracle#env-variables).
 
-#### Environment variables
+### Lido Locator address
 
-The oracle daemon requires the following environment variables:
+**Mainnet**  
+**0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb**
```

Comment:
Maybe make it clickable?

Reply:
Fixed

Reply:
I mean adding link to etherscan is a good option here